targets/ipq40xx-generic

```diff
@@ -96,6 +96,21 @@ device('gl.inet-gl-b1300', 'glinet_gl-b1300', {
 device('linksys-ea6350v3', 'linksys_ea6350v3')
 
 
+-- Meraki
+
+device('meraki-mr33', 'meraki_mr33', {
```

Comment:
The first reference is effectively matching the string generated from [this line](https://github.com/openwrt/openwrt/blob/master/target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4029-mr33.dts#L7) in OpenWrt.

Why would you use `meraki-mr33` instead here?
I'd suggest changing it here and removing the manifest alias below.


```suggestion
device('meraki-mr33-access-point', 'meraki_mr33', {
```

Reply:
Oh I didn't knew that it is the name in kebab-case - i thought it was just line 8 with a dash :laughing: 

https://github.com/openwrt/openwrt/blob/master/target/linux/ipq40xx/files/arch/arm/boot/dts/qcom-ipq4029-mr33.dts#L8